lib/esp32/espnow.toit

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2022 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the lib/LICENSE file.
+
+STATION ::= 0 // Wi-Fi station mode
+SOFTAP  ::= 1 // Not support yet
+
+check_mac_ address/ByteArray:
+  if address.size != 6:
+      throw "ESP-Now MAC address length must be 6 bytes"
+
+check_key_ key/ByteArray:
+  if key.size != 16:
+      throw "ESP-Now key length must be 16 bytes"
+
+check_channel_ channel/int:
```

Comment:
Remove this unused function.
(Probably add this check to `add_peer` ?)

Reply:
Moving this to `add_peer` is better, I would like to check this in toit side.